### DISCUSSION TITLE
Comparing Main to update-game does not provide a create pull request button

### DISCUSSION THREAD

**Discussion by CCSNH-NicolePhillips** [skills/.github#383] [Review pull requests]
Following the steps comparing main as base and update-game the 'Create pull request' button disappears and is replaced with the following text...
There isn’t anything to compare.
main and update-game are entirely different commit histories.
Underneath we can see the changes. My students are hung up on this project and so am I.  
Please advise.

**Comment by sinsukehlab** [comment]
I can see the **Create pull request** button though. Could you attach a screenshot or something?
https://github.com/CCSNH-NicolePhillips/skills-review-pull-requests/compare/main...update-game
